src/domain/app/layout/PageWrapper.tsx

```diff
@@ -27,17 +30,20 @@ const PageWrapper: React.FC<Props> = ({
   const translatedTitle =
     title !== 'appName' ? `${t(title)} - ${t('appName')}` : t('appName');
 
-  const path = isClient
-    ? window.location.pathname.replace(`/${locale}/`, '')
-    : '';
-
   const image = DEFAULT_SOME_IMAGE;
 
   const openGraphProperties: { [key: string]: string } = {
     image: image,
     title: translatedTitle,
   };
 
+  if (isClient) {
+    host = `${window.location.protocol}//${window.location.hostname}`;
+    url = window.location.pathname;
+  }
```

Comment:
Could we get rid of all the prop drillin and only use this client side? Then on server side we could have the Html-tsx implementation and on client-side override that here?

Reply:
Tested this and it just generates second canonical url tag

Reply:
Not sure about this but now we can get the Uri components from react context